geode-core/src/main/java/org/apache/geode/internal/cache/partitioned/ClearPRMessage.java

```diff
@@ -179,39 +169,31 @@ protected boolean operateOnPartitionedRegion(ClusterDistributionManager distribu
     return false;
   }
 
-  public boolean doLocalClear(PartitionedRegion region) throws ForceReattemptException {
+  public int getBucketId() {
+    return this.bucketId;
+  }
+
+  public boolean doLocalClear(PartitionedRegion region, int bucketId)
```

Comment:
The `int bucketId` in the parameter list is never used.

Reply:
Fixed.